app/client/src/components/pages/Community.Tabs.Protect.js

```diff
@@ -1196,44 +1196,41 @@ function Protect() {
-                                      plan.split('</a>')[0] + '</a>';
-                                    const title = getTitleFromMarkup(markup);
-                                    const planUrl = getUrlFromMarkup(markup);
-                                    if (!title || !planUrl) return null;
-                                    return { url: planUrl, title: title };
-                                  });
+                                  item.watershedPlans
+                                    ?.split(',')
+                                    .map((plan) => {
+                                      const markup =
+                                        plan.split('</a>')[0] + '</a>';
+                                      const title = getTitleFromMarkup(markup);
+                                      const planUrl = getUrlFromMarkup(markup);
+                                      if (!title || !planUrl) return null;
+                                      return { url: planUrl, title: title };
+                                    });
                                 // remove any plans with missing titles or urls
                                 const filteredProtectionPlans =
-                                  protectionPlans &&
-                                  protectionPlans.filter(
-                                    (plan) => plan && plan.url && plan.title,
+                                  protectionPlans?.filter(
+                                    (plan) => plan?.url && plan.title,
                                   );
 
                                 const protectionPlanLinks =
-                                  filteredProtectionPlans &&
-                                  filteredProtectionPlans.length > 0
-                                    ? filteredProtectionPlans.map(
-                                        (plan, index) => {
-                                          if (plan && plan.url && plan.title) {
-                                            return (
-                                              <div key={index}>
-                                                <a
-                                                  href={plan.url}
-                                                  target="_blank"
-                                                  rel="noopener noreferrer"
-                                                >
-                                                  {plan.title}
-                                                </a>
-                                              </div>
-                                            );
-                                          }
-                                          return false;
-                                        },
-                                      )
+                                  filteredProtectionPlans?.length > 0
+                                    ? filteredProtectionPlans.map((plan) => {
```

Comment:
Good call, removing that was a mistake. I was thinking it might work to use the URL as a key instead, but that's a bit presumptuous, I think an index is fine.